src/routes/user/[name=validname]/index.js

```diff
@@ -35,7 +35,17 @@ async function getAverage(name) {
                 name: name
             }
         }
-    } else if (data.followers > data.average) {
+    } else if (data.average < 4 && data.followers > 20) {
+        return {
+            body: {
+                result: 'botted?',
+                avg: data.average,
+                followers: data.followers,
+                diff: diff,
+                name: name
+            }
+        }
+    } else if (data.followers > (data.average * 2) ) {
```

Comment:
not sure about this

Reply:
why can't it just do `followers > average`? although I guess multiplying by 2 does give it a higher bar before you're overrated